extensions/amp-list/0.1/amp-list.js

```diff
@@ -217,6 +217,10 @@ export class AmpList extends AMP.BaseElement {
       this.element.warnOnMissingOverflow = false;
     }).then(() => {
       this.adjustContainerForLoadMoreButton_();
+      listen(this.getLoadMoreService_().getLoadMoreFailedClickable(),
```

Comment:
nit: is an unlistener needed?

Reply:
So you would want to `unlisten` this on `unlayout`, but `<amp-list>` doesn't have `unlayoutCallback` defined. If you `unlisten`-ed this on end, you would have to re-`listen` them after changing `src`. I don't think it's currently necessary to unlisten them, but we could.